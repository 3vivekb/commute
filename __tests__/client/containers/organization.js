/* global describe, expect, it */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'
import { Provider } from 'react-redux'

import { makeMockStore, mockStoreData } from '../../../test-utils/mock-store.js'

import Organization from '../../../client/containers/organization'

const mockStore = makeMockStore(mockStoreData)

describe('Container > Organization', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <Organization />
      </Provider>
    )
    expect(mountToJson(tree.find(Organization))).toMatchSnapshot()
  })
})
