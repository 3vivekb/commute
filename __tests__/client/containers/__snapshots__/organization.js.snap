exports[`Container > Organization renders correctly 1`] = `
<Connect(Organization)>
  <Organization
    _id="1"
    dispatch={[Function]}
    groups={Array []}
    name="Mock Organization"
    sites={Array []}>
    <div>
      <h1>
        <pure(Icon)
          type="object-ungroup">
          <i
            className="fa fa-object-ungroup fa-fw " />
        </pure(Icon)>
      </h1>
      <p>
        Below are this organizations sites and commuter groups.
      </p>
      <h1>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/organizations/1/sites/create">
          <a
            onClick={[Function]}
            style={Object {}}>
            <pure(Icon)
              type="building">
              <i
                className="fa fa-building fa-fw " />
            </pure(Icon)>
          </a>
        </Link>
      </h1>
      <p>
        A site is a location of a building or new address that you want to use as the centerpoint of your commutes.
      </p>
      <ul>
        <ListSites
          sites={Array []}>
          <p>
            No sites found
          </p>
        </ListSites>
      </ul>
      <h1>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/organizations/1/groups/create">
          <a
            onClick={[Function]}
            style={Object {}}>
            <pure(Icon)
              type="users">
              <i
                className="fa fa-users fa-fw " />
            </pure(Icon)>
          </a>
        </Link>
      </h1>
      <ul>
        <ListGroups
          groups={Array []}>
          <p>
            No groups found
          </p>
        </ListGroups>
      </ul>
    </div>
  </Organization>
</Connect(Organization)>
`;
